ads/google/a4a/utils.js

```diff
@@ -156,6 +171,9 @@ export function googleAdUrl(
         {name: 'brdim', value: additionalDimensions(win, viewportSize)},
         {name: 'isw', value: viewportSize.width},
         {name: 'ish', value: viewportSize.height},
+        // TODO(levitzky) Look into expandable support.
+        {name: 'ea', value: '0'},
+        {name: 'pfx', value: pfx},
```

Comment:
I don't mind documenting these in the repo, but the other params are not documented, and I'm not quite sure what should be kept internal.